rancher/dashboard
# `vee-validate` experiment

This PR is used to evaluate `vee-validate` validations library and eventually apply to Rancher UI.

Contributes to #8365

### How to test it ###

1. Go to local cluster -> Workloads -> Jobs
2. Create a new `Job`
3. Validation should works correctly:
    - Rules:
        - name is required;
        - container name is required;
        - container name should contains numbers 
    - Job `name` field is validated by form validation
    - Container name is validated at model level. Unreported error should appears on top of the form 
    - Create button should be disabled
    - Tab errors should be visible


Signed-off-by: Francesco Torchia <francesco.torchia@suse.com>
